src/GitHub.VisualStudio.UI/Styles/ThemeDesignTime.xaml

```diff
@@ -4,7 +4,7 @@
   xmlns:PresentationOptions="http://schemas.microsoft.com/winfx/2006/xaml/presentation/options">
 
   <ResourceDictionary.MergedDictionaries>
-    <ResourceDictionary Source="pack://application:,,,/GitHub.VisualStudio.UI;component/Styles/ThemeBlue.xaml" />
+    <ResourceDictionary Source="pack://application:,,,/GitHub.VisualStudio.UI;component/Styles/ThemeLight.xaml" />
```

Comment:
This was me checking out other themes. I'll change this back to make the diff look better by one line 😉